cogboard-webapp/src/constants/index.js

```diff
@@ -52,3 +54,16 @@ export const INITIAL_BOARD_PROPS = {
   autoSwitch: false,
   switchInterval: 0
 };
+
+export const NOTIFICATIONS = {
+  LOGIN: userRole => ({
+    type: 'success',
+    message: `Logged in as ${userRole}`,
+    duration: 3000
+  }),
+  LOGOUT: userRole => ({
+    type: 'info',
+    message: `${capitalize(userRole)} logged out`,
+    duration: 3000
```

Comment:
we could have this as a named Constant **3_SECONDS**

Reply:
I am not sure if it is necessary because 3_SECONDS will always be 3000 and 3000 has clear meaning, but maybe I am wrong.